src/python/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -216,6 +223,13 @@ def fuzz(self, target_path, options, reproducers_dir, max_time):
         artifact_prefix=reproducers_dir,
         extra_env=options.extra_env)
 
+    if (not environment.get_value('USE_MINIJAIL') and
```

Comment:
This means we don't get results for OSS-Fuzz, should try finding some workaround ?

Reply:
I'm not sure what the best workaround here. Longer term we'll move from minijail to something more lightweight like chroot which should make this issue disappear.. but in any case this is a limitation that has always existed.